setup.sh

```diff
@@ -41,11 +41,14 @@ elif [[ $HOSTNAME == *"tamsa2"* ]]; then
 elif [[ $HOSTNAME == *"knu"* ]]; then
 
   echo "@@@@ Working on KNU"
-  export SKFlatRunlogDir="/u/user/$USER/scratch/SKFlatRunlog/"
-  export SKFlatOutputDir="/u/user/$USER/scratch/SKFlatOutput/"
+  export SKFlatRunlogDir="/d0/scratch/$USER/SKFlatRunlog/"
+  export SKFlatOutputDir="/d0/scratch/$USER/SKFlatOutput/"
 
 fi
 
+mkdir -p $SKFlatRunlogDir
+mkdir -p $SKFlatOutputDir
```

Comment:
Maybe these two lines are not needed? Or should we make this "if dir does not exist 'mkdir'"?

Reply:
I guess SKFlat.py script will make these directory later. But there will be no harm adding these two lines.  
And I think checking existing directory is not needed with -p option.